server/controllers/api.py

```diff
@@ -923,6 +932,19 @@ def post(self, user, backup_id):
 
         return self.schema.store_comment(user, backup)
 
+    @marshal_with(schema.get_fields)
+    def get(self, user, backup_id):
+        backup = models.Backup.query.get(backup_id)
+        if not backup:
+            if user.is_admin:
+                restful.abort(404)
+            else:
+                restful.abort(403)
+        if not models.Backup.can(backup, user, "view"):
+            restful.abort(403)
+        return {"comments": [backup.comments]}
```

Comment:
Do you need to make this a list? 

> [backup.comments]

I think it's already a list because of the relationship setup in sqlalchemy - so wrapping it in a list again will get you a nested list.